docs/source/user-guide/example-usage.md

```diff
@@ -228,7 +228,7 @@ codegen-units = 1
 
 Then, in `main.rs.` update the memory allocator with the below after your imports:
 
-```rust,ignore
+```rust ,ignore
```

Comment:
I just put a space before the comma. Thankfully, this works with both Rust and Sphinx😎